components/camel-aws/camel-aws-secrets-manager/src/main/java/org/apache/camel/component/aws/secretsmanager/vault/CloudTrailReloadTriggerTask.java

```diff
@@ -349,7 +349,6 @@ public void run() {
                                     if (isReloadEnabled()) {
                                         LOG.info("Update for AWS secret: {} detected, triggering CamelContext reload", name);
                                         triggerReloading = true;
-                                        message.receiptHandle();
```

Comment:
@oscerd just wonder if this method did some that was needed or it can be safely removed

Reply:
ah its a noop